harness/determined/keras/_tf_keras_context.py

```diff
@@ -40,6 +49,61 @@ def __init__(self, env: det.EnvContext, hvd_config: horovod.HorovodContext):
         self._wrapped_optimizers = []  # type: List[tf.keras.optimizers.Optimizer]
         self._compiled_optimizer = None  # type: Optional[tf.keras.optimizers.Optimizer]
 
+        # The following attributes may be configured via configure_fit().  Defaults match the
+        # normal keras.fit() defaults.
+        self._fit_verbose = True
+        self._fit_class_weight = None
+        self._fit_workers = 1
+        self._fit_use_multiprocessing = False
+        self._fit_max_queue_size = 10
+
+    def configure_fit(
+        self,
+        verbose: Optional[bool] = None,
+        class_weight: Any = _arg_not_provided,
+        workers: Optional[int] = None,
+        use_multiprocessing: Optional[bool] = None,
+        max_queue_size: Optional[bool] = None,
```

Comment:
non-blocking: allow users to specify `validation_steps` also?

Reply:
let's add it in a future change, after reworking the data loading code.

Reply:
sounds good, do you mind filing a jira so we remember

Reply:
DET-4529